com.albiononline.AlbionOnline.json

```diff
@@ -0,0 +1,82 @@
+{
+    "app-id": "com.albiononline.AlbionOnline",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "albion-online",
+    "tags": ["proprietary"],
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+	"--persist=.albiononline",
+	"--extra-data=albion-online-setup:6a27443b66386e1f2f037667ab9b3c6eef50b5f08105289d43c253708fcdc4ce:68848539::https://live.albiononline.com/clients/20170616104311/albion-online-setup"
+    ],
+    "modules": [
+	{
+            "name": "udev",
+            "config-opts": ["--disable-hwdb", "--disable-logging", "--disable-gudev",
+                            "--disable-introspection", "--disable-keymap", "--disable-mtd_probe"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.bz2",
+                    "sha256": "4c7937fe5a1521316ea571188745b9a00a9fdf314228cffc53a7ba9e5968b7ab"
+                }
+            ]
+        },
+	{
+	    "name": "albion-online",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"gcc -W -Wall splitelf.c -o splitelf",
+		"install -D splitelf /app/bin/splitelf",
+		"install -D albion-online /app/bin/albion-online",
+		"install -D apply_extra /app/bin/apply_extra",
+		"install -Dm644 com.albiononline.AlbionOnline.appdata.xml /app/share/appdata/com.albiononline.AlbionOnline.appdata.xml",
+		"install -Dm644 com.albiononline.AlbionOnline.desktop /app/templates/com.albiononline.AlbionOnline.desktop"
+	    ],
+	    "sources": [
+		{
+		    "type": "file",
+		    "path": "splitelf.c"
+		},
+		{
+		    "type": "script",
+		    "dest-filename": "albion-online",
+		    "commands": [
+			"if ! [ -x \"$HOME/.albiononline/launcher/Albion-Online\" ]; then",
```

Comment:
Personally I find it more readable to have any `script` longer than 3 or 4 lines in separate file. Feel free to ignore this though…